cmd/commands/client/options.go

```diff
@@ -60,6 +64,13 @@ func ParseArguments(args []string) (options CommandOptions, err error) {
 		"Run an interactive CLI based Mysterium UI",
 	)
 
+	flags.StringVar(
+		&options.DiscoveryAPIAddress,
+		"discovery-address",
+		server.MysteriumApiUrl,
```

Comment:
`client` shouldn't rely on `server` for options - they should be independant.
Can we extract this param into common class in `cmd` package, i.e. `cmd.CommonOptions`?

Reply:
This is gotcha :) and example of bad naming. "server" package has nothing to do with server functionality, it's simply part of discovery api functionality.

Reply:
true, moved to cmd package.